oidc-exchange.py

```diff
@@ -179,7 +214,14 @@ def _get(name: str) -> str:  # noqa: WPS430
 try:
     oidc_token = id.detect_credential(audience=oidc_audience)
 except id.IdentityError as identity_error:
-    die(_TOKEN_RETRIEVAL_FAILED_MESSAGE.format(identity_error=identity_error))
+    if event_is_third_party_pr():
+        die(
+            _TOKEN_RETRIEVAL_FAILED_FORK_PR_MESSAGE.format(
+                identity_error=identity_error,
+            ),
+        )
+    else:
+        die(_TOKEN_RETRIEVAL_FAILED_MESSAGE.format(identity_error=identity_error))
```

Comment:
I prefer having a single die call with only the variable part being conditional. This helps with reducing code nesting and — by extension — clarity.
```suggestion
    cause_msg_tmpl = (
        _TOKEN_RETRIEVAL_FAILED_FORK_PR_MESSAGE if event_is_third_party_pr()
        else _TOKEN_RETRIEVAL_FAILED_MESSAGE
    )
    for_cause_msg = cause_msg_tmpl.format(identity_error=identity_error)
    die(for_cause_msg)
```